internal/server/rule_init.go

```diff
@@ -230,22 +231,34 @@ func scheduleCronRule(now time.Time, options *def.RuleOption) (bool, error) {
 }
 
 func startCPUProfiling(ctx context.Context) error {
-	if err := cpuprofile.StartProfilerAndAggregater(ctx, time.Duration(1000)*time.Millisecond); err != nil {
+	cpuprofile.EnableWindowAggregator(30)
+	if err := cpuprofile.StartCPUProfiler(ctx, time.Duration(1000)*time.Millisecond); err != nil {
 		return err
 	}
-	receiveChan := make(chan *cpuprofile.DataSetAggregate, 1024)
-	cpuprofile.RegisterTag("rule", receiveChan)
 	go func(ctx context.Context) {
+		ticker := time.NewTicker(time.Duration(15) * time.Second)
+		defer ticker.Stop()
 		for {
 			select {
 			case <-ctx.Done():
 				return
-			case data := <-receiveChan:
-				// TODO: support query in future
-				conf.Log.Debugf("cpu profile data: %v", data)
+			case <-ticker.C:
+				data := cpuprofile.GetWindowData()
+				if data == nil {
+					continue
+				}
+				ruleUsage, ok := data["rule"]
+				if !ok {
+					continue
+				}
+				for labelValue, t := range ruleUsage.Stats {
+					fmt.Println(labelValue, t)
```

Comment:
Remove this? At least don't use fmt.Printxx

Reply:
sorry, i forgot to remove it(for debug)🥺